### PR TITLE
clarifications, corrections and improvements to lunchbytes page

### PR DESCRIPTION
**Pull request by davidwyld**

### CODE REVIEW

File: pages/community/lunch-bytes.md [hunk 1]
```diff
@@ -16,29 +16,36 @@ use/manage research data and
 use/manage research infrastructure.
 We hope through these talks we will come together as a community to discuss best practices and useful methods/tools.
 
-For those familiar with the [RSE seminar series](/community/seminars/), LunchBytes has replaced it but is different in flavour:
-each session features several related short talks on a theme (each being ~10 mins) rather than one longer talk.
+Each LunchBytes session features several related short talks on a theme (each being ~10 mins) rather than one longer talk - followed by a dicussion session.
```
Comment (willfurnass) [inline pages/community/lunch-bytes.md]: Agree that it's not worth making comparisons to our seminar series any more.

s/features/typically features/

sp of 'dicussion'

File: pages/community/lunch-bytes.md [hunk 2]
```diff
@@ -16,29 +16,36 @@ use/manage research data and
...
 
+### Suggesting LunchBytes topics
 Please add requests/offers of LunchBytes talks to [this JamBoard](https://jamboard.google.com/d/1-51cRf0pwZl8O10CnLeJGAqKcnbww-QGaYjszFK-H38/) (using Sticky Notes):
- - For requests: a brief note for why you think this would be useful would be helpful
- - For offers: please note your email address in addition to your talk idea
+ - For __requests__: a brief note for why you think this would be useful would be helpful
+ - For __offers__: please note your email address in addition to your talk idea
  - If you want to up-vote a request/offer use the Pen tool to annotate a Sticky Note with **+1**.
 
+## Recordings of previous LunchBytes
```
Comment (willfurnass) [inline pages/community/lunch-bytes.md:42]: Do we want to start adding links to recordings here in addition to the per-session pages?  Also, if we really do want to do this do we need to update the operations guide?

**Review by willfurnass — CHANGES_REQUESTED** [review]

File: pages/community/lunch-bytes.md (hunk 2 above)
Comment (davidwyld) [reply pages/community/lunch-bytes.md:42]: I'm not sure we want all of them here  - maybe we do? I at least wanted a few as a way of making them more visible.

Comment (willfurnass) [reply pages/community/lunch-bytes.md:42]: Okay, seems reasonable, provided we have a prompt to remind us to append to this list per event.

**Review by willfurnass — APPROVED** [review]

File: pages/community/lunch-bytes.md (hunk 2 above)
Comment (davidwyld) [reply pages/community/lunch-bytes.md:42]: I'll add something to the guidance

**Pull request merged**